### DISCUSSION TITLE
Shure mxn5

### DISCUSSION THREAD

**Discussion by testhassio** [bitfocus/companion#3099] [Q&A] [answered]
Hello,

I have four of these speakers that I am trying to control with a Stream Deck +, I can turn the volume gain up when turning right with the rotary action, but cannot for the life of me figure out why I can't turn the volume down by turning the dial to the left. I can't even get the volume to go down when i choose a standard button to lower the gain. If someone could point me in the right direction i would be grateful.

**Comment by dnmeid** [comment]
What action and options do you use?

**Reply by testhassio** [reply]
I selected the Rotary Action, with Rotate Left. Channel is set to Inputs Summed, with Gain Increment set to -1


![Screenshot 2024-10-23 112245](https://github.com/user-attachments/assets/57a4a42b-1f58-4c4c-9622-97f42a31b116)

**Comment by dnmeid** [comment, marked as answer]
For the records: there has been a bug and it is resolved. https://github.com/bitfocus/companion-module-shure-mxn5/issues/2
